### DISCUSSION TITLE
🇺🇳🏭📋 Welcome to discussions for the UNDP Carbon Credit Registry!

### DISCUSSION THREAD

**Discussion by mbelinsky** [undp/carbon-registry#17] [Announcements]
## 👋 Welcome!
  We’re using GitHub Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

 🤖 **Digital Public Good** This software adheres to privacy and other applicable laws and best practices, do no harm, and help attain the SDGs & contribute to sustainable development. Together we can discuss how to keep ourselves compliant in this mission.

  📣 **Language Localization** Let's make sure the Carbon Registry has as many languages as possible. We're kicking off in English with some Spanish and French. That's just the start! We can coordinate here to minimize duplication. 

  🎨 **Design Customization** National registries mean national priorities. From color templates to images and maps, we can coordinate questions and discussions here.

  ➡️ **Convert issues to discussions** You can do this either individually or bulk by labels. Looking at you, issues labeled “question” or “discussion”.

**Comment by renepostnl** [comment]
Hi Mark, really interesting stuff, we need more transparency in the carbon domain! Is this live somewhere?

**Comment by timschloe** [comment]
Just a heads up, Installing the Docker on Debian 11 gives me these two warnings:

![image](https://user-images.githubusercontent.com/6918041/229568655-21892f66-222d-48c6-9fc9-c27687360d93.png)

![image](https://user-images.githubusercontent.com/6918041/229568843-b4d04a15-aafe-4ea3-b319-5147dcdfa324.png)

Also, I just discovered that our minimum System Requirements are 4 GB RAM for the install.
